backend/routes/post.js

```diff
@@ -112,13 +119,16 @@ router.patch('/:_pid/:_cid', (request, response, next) => {
 
 // DELETE SPECIFIC COMMENT
 router.delete('/:_pid/:_cid', (request, response, next) => {
-  console.log("test");
   request.comment.remove(function(err) {
+    if (err) return next(err);
+    request.post.comments.pull(request.params._cid);
```

Comment:
Where is this `pull` method comming from because I don't think its part of js

Reply:
It's a Mongoose method.